documentapi/src/main/java/com/yahoo/documentapi/messagebus/protocol/AsyncInitializationPolicy.java

```diff
@@ -100,6 +105,7 @@ public void run() {
         try {
             init();
         } catch (Exception e) {
+            log.warning("Init threw exception" + exceptionMessageWithTrace(e));
```

Comment:
You don't need to do this for logging - just do log.log(Level.WARNING, "Could not initialize policy", e);

Reply:
Great. Changed it.